lib/osf-components/addon/components/file-actions-menu/styles.scss

```diff
@@ -25,4 +25,12 @@
 .DropdownItem {
     composes: FakeLink from '../button/styles.scss';
     padding: 8px 0 0;
+
+    div:nth-of-type(3) {
+        padding-left: 2px;
+    }
+}
+
+.metadata-link {
+    padding-top: 6px;
```

Comment:
It's odd you had to do specific formatting for this. I guess because everything else is a button with fake link styling rather than an OSF Link? 

Is it possible to put the padding-left in with a style attached to a specific item rather than nth-of-type? I can see us possibly re-arranging this menu, and I wouldn't want to have to keep changing that selector each time if we don't have to.